### ISSUE TITLE
JuliaToGAP cannot convert Julia ranges to GAP lists

### ISSUE THREAD

**Issue by mohamed-barakat** [oscar-system/GAP.jl#439]
```
gap> JuliaEvalString( "1:3" );
<Julia: 1:3>
gap> JuliaToGAP( IsList, JuliaEvalString( "1:3" ) );
Error, Constructor: the first argument must be a filter (not a JuliaObject)
not in any function at *stdin*:3
type 'quit;' to quit to outer loop
brk>
```

**Comment by mohamed-barakat** [comment]
I lost the overview of current issues. Sorry if this one is covered by an existing issue.

**Comment by ThomasBreuer** [comment]
As far as I see, this is new.
I will create a pull request for this and #438.

**Comment by ThomasBreuer** [comment]
done because of #444

**Comment by fingolfin** [comment]
@ThomasBreuer you can write messages like `closes #12345` into the description of your PRs; this way, when they are merged, the relevant issues (or other PRs) are automatically closed; see https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords
